CHANGELOG.md

```diff
@@ -10,8 +10,16 @@ internal API changes are not present.
 Main (unreleased)
 -----------------
 
+v0.28.0-rc.0 (2022-09-21)
+-------------------------
+
 ### Features
 
+- Introduce Grafana Agent Flow, an experimental "programmable pipeline" runtime
+  mode which improves how to configure and debug Grafana Agent by using
+  components. (@captncraig, @marctc, @mattdurham, @rfratto, @rlankfo,
+  @tpaschalis)
```

Comment:
Agh, I meant to add @karengermond into this list but I was in a rush. I think it's too late to do it for the release candidate since we already published it, but I'm setting a reminder to add her in for the stable release.